### PR TITLE
Get travis passing

### PR DESCRIPTION
**Pull request by briancappello**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 python:
-  - "2.6"
   - "2.7"
-  - "3.2"
   - "3.3"
   - "3.4"
   - "3.5"
 os:
   - linux
-  - osx
```
Comment (briancappello) [inline .travis.yml]: See https://github.com/travis-ci/travis-ci/issues/2312

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,22 @@
 language: python
 python:
-  - "2.6"
```
Comment (briancappello) [inline .travis.yml]: OrderedDict is 2.7+

File: .travis.yml [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
   - "2.7"
-  - "3.2"
```
Comment (briancappello) [inline .travis.yml]: Pandas is 3.3+

**Comment by mrjbq7** [comment]
Hah, thank you! I was starting to work on this but got distracted yesterday!

**Comment by femtotrader** [comment]
Closes #104

**Pull request merged**